src/Core/src/Handlers/Picker/PickerHandler.iOS.cs

```diff
@@ -223,15 +226,16 @@ public void Disconnect(MauiPicker platformView)
 			}
 
 #if MACCATALYST
-			bool OnShouldBeginEditing (UITextField textField)
+			bool OnShouldBeginEditing(UITextField textField)
 			{
 				if (Handler is not PickerHandler handler)
```

Comment:
Mayne just bail out if virtual view is null like the end editing instead of the null check but still shopping the popup.